gravitee-apim-gateway/gravitee-apim-gateway-tests-sdk/src/main/java/io/gravitee/apim/gateway/tests/sdk/runner/GatewayRunner.java

```diff
@@ -454,8 +506,34 @@ private void ensureMinimalRequirementForConnectors(Map<String, ConnectorPlugin>
         connectors.putIfAbsent("connector-http", ConnectorBuilder.build("connector-http", HttpConnectorFactory.class));
     }
 
-    private Api loadApiDefinition(String apiDefinitionPath) throws IOException {
-        return loadResource(apiDefinitionPath, Api.class);
+    private void registerEntrypoints(GatewayTestContainer container) {
+        final EntrypointConnectorPluginManager entrypointPluginManager = container.applicationContext().getBean(EntrypointConnectorPluginManager.class);
+
+        Map<String, EntrypointConnectorPlugin<?>> entrypointsMap = new HashMap<>();
+        testInstance.configureEntrypoints(entrypointsMap);
+        ensureMinimalRequirementForEntrypoints(entrypointsMap);
+        entrypointsMap.forEach((key, value) -> entrypointPluginManager.register(value));
+    }
+
+    private void ensureMinimalRequirementForEntrypoints(Map<String, EntrypointConnectorPlugin<?>> entrypoints) {
+        entrypoints.putIfAbsent("entrypoint-sse", EntrypointBuilder.build("entrypoint-sse", SseEntrypointConnectorFactory.class));
```

Comment:
Here we are using `entrypoint-sse` but the real id of this plugin is `sse` only, I think we should align but I don't know which one to update 🤔

Reply:
You should rely on the plugin aka the real id. 
So i would modify this line and the json of the api declaring this entrypoint accordingly to this modification

Reply:
Ok will do this!